engine/src/main/java/org/terasology/rendering/dag/nodes/BackdropNode.java

```diff
@@ -53,14 +57,13 @@
     public void initialise() {
         super.initialise();
         playerCamera = worldRenderer.getActiveCamera();
-        requireFBO(new FBOConfig("sceneOpaque", 1.0f, FBO.Type.HDR).useDepthBuffer().useNormalBuffer().useLightBuffer().useStencilBuffer());
-        requireFBO(new FBOConfig("sceneReflectiveRefractive", 1.0f, FBO.Type.HDR).useNormalBuffer());
+        requireDynamicFBO(new FBOConfig(REFRACTIVE_REFLECTIVE_URN, 1.0f, FBO.Type.HDR).useNormalBuffer());
     }
 
     @Override
     public void process() {
         PerformanceMonitor.startActivity("rendering/backdrop");
-        sceneOpaque = frameBuffersManager.getFBO("sceneOpaque");
+        sceneOpaque = dynamicFBM.getFBO(DefaultDynamicFBOs.ReadOnlyGBuffer.getResourceUrn());
```

Comment:
In line with a previous comment, this is one of those situation in which I'm wondering: shouldn't the variable sceneOpaque already have an up to date FBO? Either obtained on initialization or after an update triggered by the FBM regenerating the FBOs? In short, this line shouldn't be necessary here in process().